server/src/graphql-types/EventReminder.ts

```diff
@@ -0,0 +1,23 @@
+import { Field, Int, ObjectType } from 'type-graphql';
+import { EventWithEverything, User } from '.';
+
+@ObjectType()
+export class EventReminder {
+  @Field(() => Int)
+  user_id: number;
+
+  @Field(() => Int)
+  event_id: number;
+
+  @Field(() => Date)
+  remind_at: Date;
+
+  @Field(() => User)
+  user: User;
+
+  @Field(() => EventWithEverything)
+  event: Omit<EventWithEverything, 'sponsors' | 'rsvps'>;
+
+  @Field(() => Boolean)
+  notified: boolean;
```

Comment:
```suggestion
  notifying: boolean;
```
To keep in sync with the db.